src/skin/popup.html

```diff
@@ -114,6 +114,10 @@ <h2><span class="i18n_name"></span></h2>
 
 </div>
 
+<div id="youtube-message-container" style="display:none">
+  <div id="youtube-message"></div>
```

Comment:
Let's try making this message more legible. Bigger text?

Reply:
I made it 2px bigger. I can make it bigger (and/or change the background/text color) but it'll spill onto 3 lines.
<img width="551" alt="Screenshot 2024-06-10 at 5 17 10 PM" src="https://github.com/EFForg/privacybadger/assets/40436018/b01460cc-9375-4163-801a-290c593147c7">

Reply:
>(Bonus Panic UI Tip: people can’t/won’t read more than three (3) lines of static descriptive text.)

-- https://cabel.com/2023/03/27/apple-passwords-deserve-an-app/